dist/js-joda.d.ts

```diff
@@ -1163,18 +1163,73 @@ declare namespace JSJoda {
         static from(temporal: TemporalAccessor): YearMonth
 
         static now(zoneIdOrClock?: ZoneId|Clock): YearMonth
+        static now0(): YearMonth
+        static nowZoneId(zone: ZoneId): YearMonth
+        static nowClock(clock: Clock): YearMonth
 
         static of(year: number, monthOrNumber: Month|number): YearMonth
+        static ofNumberMonth(year: number, month: Month): YearMonth
+        static ofNumberNumber(year: number, month: number): YearMonth
 
         static parse(text: string, formatter?: DateTimeFormatter): YearMonth
+        static parseString(text: string): YearMonth
+        static parseStringFormatter(text: string, formatter: DateTimeFormatter): YearMonth
+
+        constructor(year: number, month: number)
+
+        minus(amount: TemporalAmount): YearMonth
+        minus(amountToSubtract: number, unit: TemporalUnit): YearMonth
```

Comment:
i am not a typescript user, but what would be the advantage of these split definitions, instead of the optional/overloaded definitions that we had in the file originally? To me it seems more clear with the overloaded definition?

Reply:
The JavaScript API as I see it is basically:

* You must specify one or two parameters
* If you specify one parameter then it's treated as `TemporalAmount`
* If you specify two parameters first one is `number` and the second one is `TemporalUnit`

With your original definition you could make this two mistakes:

* Pass `number` and don't specify unit, because second argument is always optional
* Pass `TemporalAmount` and `TemporalUnit` as second parameter and the method would assume that first one is `number`

After the change it will not compile and only the two correct combinations of arguments are allowed.

Reply:
True... good point. But that would mean we should change the other optional/overloaded definitions in the typescript file as well ... 

while our overloading *should* check this and cause an error at runtime, your change would allow the error to already occur at compile time... i think i like that :)